integration_tests/tracecontext/README.md

```diff
@@ -0,0 +1,7 @@
+# OpenTelemetry W3C Context Propagation tests
```

Comment:
Can you add some notes in here about the environmental requirements to have this be runnable? It looks like you'll need to be on some sort of system that supports a bourne shell, and have python3/pip3 installed.

Reply:
It'd be great if we can push a docker image with this test environment. I wouldn't block this PR on it, but then we'd even be able to run it as part of the build.

Reply:
I would have to guess that there's a stock ubuntu image out there with python3/pip3 already ready-to-go.

Reply:
updated readme and circleci config to run in docker image

Reply:
circleci config reverted